README.md

```diff
@@ -174,6 +174,36 @@ await Descope.otp.signIn(method: DeliveryMethod.email, loginId: 'desmond_c@mail.
 Note that any custom claims added via this method are considered insecure and will
 be nested under the `nsec` custom claim.
 
+## Error Handling
+
+All operations in the Descope SDK will throw a `DescopeException` in the event an error occurs.
+There are several ways to catch and handle a `DescopeException` thrown by a Descope SDK
+operation, and you can use whichever one is more appropriate in each specific use case.
```

Comment:
```suggestion
All authentication operations throw a `DescopeException` in case of a failure. There are several
ways to catch and handle a `DescopeException`, and you can use whichever one is more
appropriate in each specific use case.
```